networkx/algorithms/centrality/reaching.py

```diff
@@ -184,6 +186,9 @@ def local_reaching_centrality(G, v, paths=None, weight=None, normalized=True):
         total_weight = G.size(weight=weight)
         if total_weight <= 0:
             raise nx.NetworkXError("Size of G must be positive")
+        # self loops
+        if total_weight > 0 and len(G) == 1:
+            raise nx.NetworkXPointlessConcept("G is a single node with self loop")
```

Comment:
The [pointless concept exception](https://networkx.org/documentation/stable/reference/exceptions.html#networkx.NetworkXPointlessConcept) is (again, in principle) supposed to be used specifically when the null graph is involved.

I'd vote to use `nx.NetworkXError` instead, and to update the exception message to be more explicit:

```suggestion
            raise nx.NetworkXPointlessConcept(
                "local_reaching_centrality of a single node with self-loop not well-defined"
            )
```